test/system/700-play.bats

```diff
@@ -411,16 +411,19 @@ _EOF
 # Occasionally a remnant storage container is left behind which causes
 # podman play kube --replace to fail. This tests created a conflicting
 # storage container name using buildah to make sure --replace, still
-# functions proplery by removing the storage container.
+# functions properly by removing the storage container.
 @test "podman kube play --replace external storage" {
-    _write_test_yaml
+    _write_test_yaml command="top"
 
     run_podman play kube $TESTYAML
     # Force removal of container
     run_podman rm --force -t0 $PODCTRNAME
+    assert "$output" = "$PODCTRNAME" "Confirmation that container was removed"
+
     # Create external container using buildah with same name
     buildah from --name $PODCTRNAME $IMAGE
-    # --replace deletes the buildah container and replace it with new one
+    # --replace deletes the buildah container and replace it with new one.
+    # Prior to #20025, this would fail with "container name is in use"
     run_podman play kube --replace $TESTYAML
```

Comment:
should we call play kube first without --replace, make sure we get the right error and then do it with replace again to ensure we never regress again

Reply:
I did that, but it's not a useful test: it fails with "pod name already exists" (because the container is killed, but not the pod). I considered `kube down` but wasn't sure if that would trigger the same failure. I think the output check on `podman rm` is good enough (and of course I tested it.)

Reply:
but that check doesn't ensure that buildah created a container with the right storage name.
In theory it could use a different storage name from the cli name in the future then this is a NOP again.
Maybe do podman ps --external check before?

I do not understand at all what the point of line 418-420 is at all for the purpose of the test.

Reply:
Is this better?